coverage.rst

```diff
@@ -72,30 +72,53 @@ Using coverage.py
 One of the most popular third-party coverage tools is `coverage.py`_ which
 provides very nice HTML output along with advanced features such as
 :ref:`branch coverage <branch_coverage>`. If you prefer to stay with tools only
-provided by the stdlib then you can by :ref:`using test.regrtest
+provided by the stdlib then you can :ref:`use test.regrtest
 <coverage_by_regrtest>`.
 
-Because the in-development version of Python is bleeding-edge, it is possible
-that the latest release version of coverage.py will not work. In that case you
-should try using the in-development of coverage.py to see if it has been
-updated as needed. To do this you should clone/check out the development version
-of coverage.py::
+
+.. _install_coverage:
+
+Install Coverage
+''''''''''''''''
+
+By default, pip will not install into the in-development version of Python you 
+just built, and this built version of Python will not see packages installed 
+into your default version of Python. One option is to use a virtual environment
+to install coverage::
+
+    ./python.exe -m venv ../cpython-venv
+    source ../cpython-venv/bin/activate
+    pip install coverage
+
+You can now use python without the ./ for the rest of these instructions, as long as
+your venv is activated. For more info on venv see https://docs.python.org/3/tutorial/venv.html.
+ 
+If this does not work for you for some reason, you should try using the in-development 
+of coverage.py to see if it has been updated as needed. To do this you should 
+clone/check out the development version of coverage.py:
 
     hg clone https://bitbucket.org/ned/coveragepy
 
-Another option is to use an installed copy of coverage.py if you already have an
-installed copy. But if you do not already have it installed then it is preferred
-you use a clone of coverage.py for gathering coverage results.
+You will need to use the full path to the installation.
+
+Another option is to use an installed copy of coverage.py, if you already have an
+installed copy. For this also, you will also need to use the full path to that installation. 
```

Comment:
'For this also, you will also need [...]' -> 'For this, you will again need [...]'